<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object editor</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_bone"/>
    <addaction name="actionAdd_LOD"/>
    <addaction name="actionAdd_material"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_model_from_Blender"/>
    <addaction name="actionAdd_model_from_3DMax"/>
    <addaction name="actionAdd_model_from_obj"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_animation_from_fbx"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_ambient_light"/>
    <addaction name="actionAdd_direct_light"/>
   </widget>
   <widget class="QMenu" name="menuManipulator">
    <property name="title">
     <string>Manipulator</string>
    </property>
    <addaction name="actionSetMoveManipulator"/>
    <addaction name="actionSetRotationManipulator"/>
    <addaction name="actionSetScaleManipulator"/>
    <addaction name="separator"/>
    <addaction name="actionSetLocalOrientation"/>
    <addaction name="actionSetGlobalOrientation"/>
   </widget>
   <widget class="QMenu" name="menuStatistic">
    <property name="title">
     <string>Statistic</string>
    </property>
    <addaction name="actionStatistic"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuManipulator"/>
   <addaction name="menuStatistic"/>
  </widget>
  <action name="actionAdd_bone">
   <property name="text">
    <string>Add bone</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAdd_animation_from_fbx">
   <property name="text">
    <string>Add animation from fbx</string>
   </property>
  </action>
  <action name="actionSetMoveManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Moving</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionSetRotationManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionSetLocalOrientation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local orientation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSetGlobalOrientation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Global orientation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAdd_LOD">
   <property name="text">
    <string>Add LOD</string>
   </property>
  </action>
  <action name="actionAdd_direct_light">
   <property name="text">
    <string>Add direct light</string>
   </property>
  </action>
  <action name="actionStatistic">
   <property name="text">
    <string>Statistic window</string>
   </property>
  </action>
  <action name="actionAdd_ambient_light">
   <property name="text">
    <string>Add ambient light</string>
   </property>
  </action>
  <action name="actionSetScaleManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAdd_model_from_Blender">
   <property name="text">
    <string>Add model from Blender's fbx</string>
   </property>
  </action>
  <action name="actionAdd_model_from_3DMax">
   <property name="text">
    <string>Add model from 3DMax's fbx</string>
   </property>
  </action>
  <action name="actionAdd_model_from_obj">
   <property name="text">
    <string>Add model from obj</string>
   </property>
  </action>
  <action name="actionAdd_material">
   <property name="text">
    <string>Add material</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
